src/spatial_ref/srs.rs

```diff
@@ -7,6 +7,148 @@ use std::str::FromStr;
 
 use crate::errors::*;
 
+/// Options for [`CoordTransform::new_ex`].
+#[derive(Debug)]
+pub struct CoordTransformOptions {
+    inner: gdal_sys::OGRCoordinateTransformationOptionsH,
+}
+
+impl Drop for CoordTransformOptions {
+    fn drop(&mut self) {
+        unsafe { gdal_sys::OCTDestroyCoordinateTransformationOptions(self.inner) };
+        self.inner = ptr::null_mut();
```

Comment:
No need to do this AFAIK.

Reply:
Thought so. Smelled like a C habit; I just cargo-culted it. Removed it from `CoordTransform` as well.